guide/additional-info/reactions.md

```diff
@@ -242,26 +242,22 @@ const message = await channel.fetchMessage(data.message_id);
 
 The if statement in the middle plays an important role; it prevents us from re-emitting the event for both uncached *and* cached messages. Without this, your `messageReactionAdd` event would execute twice for a single reaction if the message was already cached.
 
-A custom emoji contains both a name and an ID, while a unicode emoji contains just a name. Since custom emoji reactions are keyed by their name:ID and unicode emoji reactions are keyed by their name, you'll have to do something like this to set the right emoji for this event:
+A custom emoji contains both a name and an ID, while a unicode emoji contains just a name. Since custom emoji reactions are keyed in a name:ID format and unicode emoji reactions are keyed by their name, you'll have to do something like this to set the right emoji for this event:
 
 ```js
-let emoji = ''
-            
-if(data.emoji.id) {
-	emoji = `${data.emoji.name}:${data.emoji.id}`
-} else {
-	emoji = data.emoji.name
-}
+const emojiKey = (data.emoji.id) ? `${data.emoji.name}:${data.emoji.id}` : data.emoji.name;
 ```
 
-We are checking for `emoji.id` because a unicode emoji won't have an ID inside the emoji object. Next we are using template literals to combine the name and the ID to create the fulle `name:ID` we need to get the custom emoji reaction.
+We are checking for `emoji.id` because a unicode emoji won't have an ID inside the emoji object. Next we are using template literals to combine the name and the ID to construct the proper key format `name:ID` we need to get the custom emoji reaction.
 
 All that's left is to fetch the actual reaction from the message and emit the event.
 
 ```js
 const reaction = message.reactions.get(emoji);
 ```
 
+<p class="tip">In the master branch/v12 reactions are keyed by their ID only, not by it's name:ID.</p>
```

Comment:
* A comma after "In the master branch/v12"
* For consistency with one of the paragraphs above, I think the last bit should be "not in a `name:ID` format" (backticks around "name:ID").